nextjs-13/src/repositories/newsletter.repository.ts

```diff
@@ -12,17 +12,29 @@ const deleteNewsletter = async (id: string, userId: string) =>
 const updateNewsletter = async (args: Prisma.NewsletterUpdateManyArgs) =>
   dbClient.newsletter.updateMany(args);
 
-const queryNewsletterByUserId = async (id: string) =>
-  dbClient.newsletter.findMany({
-    where: { userId: id },
+const queryNewsletters = async (userId: string, ids = []) => {
```

Comment:
There could be confusion on which `id` we refer to. The following would be clearer:

```suggestion
const queryNewsletters = async (userId: string, newsletterId = []) => {
```

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/121/commits/915d30ad0da55b03df03a81da84daf30b5664254#diff-7f62d1d6ff486da9e5be846b0de3dc572e91704e2e913d9aa47ab9e57090f37bR15-R32)